shared/actions/chat/inbox.js

```diff
@@ -238,7 +241,7 @@ function* _chatInboxConversationSubSaga({conv}) {
   // Wait for an idle
   yield call(onIdlePromise, 100)
   // TODO might be better to make this a put with an associated takeEvery
-  yield fork(processConversation, conv)
+  yield spawn(processConversation, conv)
```

Comment:
hmmm, maybe this should be a fork. unclear on what it means to cancel and what we want to tear down or not...